File: 가운데글자가져오기(Lv1).java
class Solution {
    public String solution(String s) {
        String[] str_arr = s.split("");
        int lenth = str_arr.length;
        
        return lenth % 2 == 0 
               ? str_arr[(lenth-1) / 2] + str_arr[((lenth-1) / 2)+1]
               : str_arr[lenth / 2];        
    }
}

File: 가운데글자가져오기(Lv1).js
function solution(s) {
    const st_arr = s.split("");
    console.log(st_arr[Math.floor((st_arr.length-1) / 2)]);
    
    return st_arr.length % 2 == 0 
           ? st_arr[Math.floor((st_arr.length-1) / 2)]+st_arr[Math.floor((st_arr.length-1) / 2)+1]
           : st_arr[Math.floor(st_arr.length / 2)];
}